### PR TITLE
Implement 'backup --no-scan'

### PR DESCRIPTION
**Pull request by baurmatt**

### CODE REVIEW

**Review by mtlynch — CHANGES_REQUESTED** [review]

File: restic/internal/backup.py [hunk 1]
```diff
@@ -17,7 +17,8 @@ def run(restic_base_command,
         exclude_files=None,
         tags=None,
         dry_run=None,
-        host=None):
+        host=None,
+        no_scan=None):
```
[the remark this answers is not in the document]
Comment (baurmatt) [reply restic/internal/backup.py]: Sure! :) Fixed.

File: docs/index.md [hunk 2]
```diff
@@ -21,6 +21,7 @@
 - `tags`: A list of tags for the new snapshot
 - `dry_run`: Set to `True` to perform just a dry run of the backup
 - `host`: Set the hostname for the snapshot manually
+- `no_scan`: Do not run scanner to estimate size of backup
```
Comment (mtlynch) [inline docs/index.md:24]: Can we update this with the new semantics?

**Review by mtlynch — CHANGES_REQUESTED** [review]

File: docs/index.md (hunk 2 above)
Comment (baurmatt) [reply docs/index.md]: Done :)

**Review by mtlynch — APPROVED** [review]
LGTM, thanks!

**Pull request merged**